pkg/infrastructure/platform/platform.go

```diff
@@ -78,6 +79,9 @@ func ProviderForPlatform(platform string, fg featuregates.FeatureGate) (infrastr
 	case libvirttypes.Name:
 		return terraform.InitializeProvider(libvirt.PlatformStages), nil
 	case nutanixtypes.Name:
+		if fg.Enabled(configv1.FeatureGateClusterAPIInstall) {
+			return clusterapi.InitializeProvider(nutanixcapi.Provider{}), nil
+		}
```

Comment:
Can you also add this to `pkg/infrastructure/platform/platform_altinfra.go`? The altinfra file is used to build a separate image that is used in CI.

Reply:
Thanks, will add this.